pyseir/models/suppression_policies.py

```diff
@@ -262,32 +294,22 @@ def generate_empirical_distancing_policy_by_state(t_list, state, future_suppress
     suppression_model: callable
         suppression_model(t) returns the current suppression model at time t
     """
-    
     county_metadata = load_county_metadata()
     counties_fips = county_metadata[county_metadata.state == state].fips.unique()
     
     if reference_start_date is None:
         reference_start_date = min([infer_t0(fips) for fips in counties_fips])
-        
-    suppression_policy_fips_args = dict(t_list=t_list, 
-                                        future_suppression=future_suppression,
-                                        reference_start_date=reference_start_date)
 
-    weight = county_metadata[county_metadata.state == state][['fips', 'total_population']] \
-                                .set_index('fips').rename(columns={'total_population': 'weight'})
-    
-    results = dict()
+    # Aggregate the counties to the state level, weighted by population.
+    weight = county_metadata.loc[county_metadata.state == state, 'total_population'].values
+    weight = weight / weight.sum()
```

Comment:
Not sure if the order of county will remain the same as that of results, better use: 
```
weight = county_metadata.set_index('fips').loc[counties_fips].values
weight = weight / weight.sum()
```

Reply:
Why not? it is a pandas dataframe so order is fixed. This method is much faster by 200ms (pandas is very slow for iteration) which when using MCMC or inference

Reply:
BTW, recommend using numpy as much as possible for things that get run in the inner optimization loops.